src/main/protobuf/soc/state.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+
+enum TurnPhase {
+    ROLL = 0;
+    BUY_TRADE_OR_END = 1;
+}
+
+message Player {
+    required string name = 1;
+    required int32 position = 2;
+}
+
+message PublicPlayerState {
+    required Player player = 1;
+    required int32 points = 2;
```

Comment:
`points` is public?

Reply:
I guess this doesnt need to be. Board points, longest road, and largest army is public